kolibri/plugins/user/assets/src/vue/index.vue

```diff
@@ -33,6 +39,18 @@
       'profile-page': require('./profile-page'),
     },
     computed: {
+      appBarTitle() {
+        if (this.pageName === PageNames.SIGN_IN) {
```

Comment:
Neither Sign Up nor Sign In use an app-bar, see `navBarNeeded()`. Sign Up kind of does but it it included within the sign-up page.